gears/src/x/types/validator.rs

```diff
@@ -0,0 +1,22 @@
+use cosmwasm_std::{Decimal256, Uint256};
+use prost::Enumeration;
+use serde::{Deserialize, Serialize};
+
+use crate::types::address::ValAddress;
+
+pub trait StakingValidator {
+    fn operator(&self) -> &ValAddress;
+    fn bonded_tokens(&self) -> &Uint256;
+    fn delegator_shares(&self) -> &Decimal256;
```

Comment:
Same as above.

Reply:
Similar case. I just followed @dmvict structures, but instead of moving them into gears added trait with getters for fields that I need